commit d580a73c3521357adcdd837c9520e14b6197c6ae
Author: Nicolas Fella <nicolas.fella@gmx.de>
Date:   Tue Dec 3 01:26:57 2019 +0100

    Port away from deprecated KIO::suggestName
    
    Test Plan: builds
    
    Reviewers: #ark, elvisangelaccio
    
    Reviewed By: #ark, elvisangelaccio
    
    Subscribers: kde-utils-devel
    
    Tags: #ark
    
    Differential Revision: https://phabricator.kde.org/D25706

diff --git b/CMakeLists.txt a/CMakeLists.txt
index abf9cc80..bf60f0e4 100644
--- b/CMakeLists.txt
+++ a/CMakeLists.txt
@@ -8,7 +8,7 @@ set (KDE_APPLICATIONS_VERSION "${KDE_APPLICATIONS_VERSION_MAJOR}.${KDE_APPLICATI
 project(ark VERSION ${KDE_APPLICATIONS_VERSION})
 
 set(QT_MIN_VERSION 5.10.0)
-set(KF5_MIN_VERSION 5.61.0)
+set(KF5_MIN_VERSION 5.44.0)
 
 find_package(ECM ${KF5_MIN_VERSION} REQUIRED NO_MODULE)
 set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules)
diff --git b/kerfuffle/jobs.cpp a/kerfuffle/jobs.cpp
index 31c9d9a1..0ec7be24 100644
--- b/kerfuffle/jobs.cpp
+++ a/kerfuffle/jobs.cpp
@@ -38,7 +38,7 @@
 #include <QTimer>
 #include <QUrl>
 
-#include <KFileUtils>
+#include <KIO/RenameDialog>
 #include <KLocalizedString>
 
 namespace Kerfuffle
@@ -440,7 +440,7 @@ void BatchExtractJob::setupDestination()
         }
 
         if (d.exists(subfolderName)) {
-            subfolderName = KFileUtils::suggestName(QUrl::fromUserInput(m_destination, QString(), QUrl::AssumeLocalFile), subfolderName);
+            subfolderName = KIO::suggestName(QUrl::fromUserInput(m_destination, QString(), QUrl::AssumeLocalFile), subfolderName);
         }
 
         d.mkdir(subfolderName);
